### PR TITLE
Changed database to catalog in org.jabref.gui.slr and org.jabref.logic.crawler

### PR DESCRIPTION
**Pull request by simar437**
In Issue #9951, it was instructed to replace the term "database" with "catalog" in the code files `org.jabref.gui.slr` and `org.jabref.logic.crawler`.

Fixes https://github.com/koppor/jabref/issues/615

### Mandatory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/resources/build.properties [hunk 1]
```diff
@@ -6,3 +6,4 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Catalogs=Create property
```
Comment (koppor) [inline src/main/resources/build.properties:9]: There was something going wrong here. This line needs to be removed, doesn't it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - In case the library contains empty entries, they are not written to disk. [#8645](https://github.com/JabRef/jabref/issues/8645)
 - The formatter `remove_unicode_ligatures` is now called `replace_unicode_ligatures`. [#9890](https://github.com/JabRef/jabref/pull/9890)
 - We improved the error message when no terminal was found [#9607](https://github.com/JabRef/jabref/issues/9607)
+- We changed database to catalog in `org.jabref.gui.slr` and `org.jabref.logic.crawler` [#9951](https://github.com/JabRef/jabref/pull/9951)
```
Comment (koppor) [inline CHANGELOG.md:64]: Changelog messages should be for the end user. We don't have a end-user documentation for the SLR functionality yet Nevertheless one could write following:

```suggestion
- In the context of the "systematic literature functionality", we changed the name "database" to "catalog" to use a separate term for online catalogs in comparison to SQL databases. [#9951](https://github.com/JabRef/jabref/pull/9951)
```

File: src/main/java/org/jabref/logic/crawler/StudyFetcher.java [hunk 3]
```diff
@@ -48,7 +48,7 @@ private QueryResult getQueryResult(String searchQuery) {
     }
 
     /**
-     * Queries all Databases on the given searchQuery.
+     * Queries all Catalogs on the given searchQuery.
```
Comment (koppor) [inline src/main/java/org/jabref/logic/crawler/StudyFetcher.java:51]: Please also fix casing when you are on it.

```suggestion
     * Queries all catalogs on the given searchQuery.
```

File: src/main/java/org/jabref/model/study/StudyDatabase.java [hunk 4]
```diff
@@ -1,7 +1,9 @@
 package org.jabref.model.study;
 
+import org.jabref.gui.slr.StudyCatalogItem;
+
 /**
- * data model for the view {@link org.jabref.gui.slr.StudyDatabaseItem}
+ * data model for the view {@link StudyCatalogItem}
```
Comment (koppor) [inline src/main/java/org/jabref/model/study/StudyDatabase.java:6]: If possible, do not add imports just for JavaDoc. -- Use the fully qualified name (FQN) for StudyCatalogItem in JavaDoc.

**Review by koppor — CHANGES_REQUESTED** [review]
Thank you for working on this. @simar437 I hope you are in the same group as @wy8881. Otherwise, it is very bad style to hijack active pull requests. It was sayd two days (!) ago that @wy8881 will continue.

Maybe, you did not realize that in GitHub terms "issue" is a thing describing that something should be fixed and "pull request" is a code contribution addressing an issue.

Nevertheless, basically, your code looks good. However, I think, you did not try it. That would have been the right action to take to really fix the underlying issue. -- The intention of this exercise is to give an insight about Software Engineering. This discipline is not only about chaninging some code, but to use proper engineering principles to improve software.

Start a SLR:

image

Still shows the old terms.

Thus, your contribution and the one of  @wy8881 need to be combined somehow.

I merged the branch of @wy8881 into yours and the UI looks good:

image

Nevertheless, there are small changes. I ask you to do it so that you learn how to use git and your IDE.

**Comment by Akshay-Gole** [comment]
Hi @koppor, Yes @simar437 and @wy8881 are in the same team. We will fix your mentioned changes as soon as possible.

**Comment by koppor** [comment]
@Akshay-Gole Thank you for the quick clarification. If possible please work on @simar437's branch (https://github.com/simar437/jabref/tree/fix-for-issue-9951) and push changes there. Then, they appear automatically in this pull request. @simar437 Can give all of you access to his repository.

**Comment by simar437** [comment]
Hi @koppor,

I made the required changes as you mentioned, thanks for your valuable insights in my PR.

File: src/main/resources/l10n/JabRef_ru.properties [hunk 5]
```diff
@@ -2225,7 +2225,7 @@ This\ entry\ type\ is\ intended\ for\ sources\ such\ as\ web\ sites\ which\ are\
 A\ single-volume\ work\ of\ reference\ such\ as\ an\ encyclopedia\ or\ a\ dictionary.=Неделимая работа или ссылка, как энциклопедия или словарь.
 A\ technical\ report,\ research\ report,\ or\ white\ paper\ published\ by\ a\ university\ or\ some\ other\ institution.=Технический отчет, исследовательский отчет, или белая книга, выпущенная институтом или другим учреждением.
 An\ entry\ set\ is\ a\ group\ of\ entries\ which\ are\ cited\ as\ a\ single\ reference\ and\ listed\ as\ a\ single\ item\ in\ the\ bibliography.=Набор записей представляет собой группу записей, которые приводятся в виде единой ссылки и перечислены в виде одного элемента в библиографии.
-Supplemental\ material\ in\ a\ "Book".\ This\ type\ is\ provided\ for\ elements\ such\ as\ prefaces,\ introductions,\ forewords,\ afterwords,\ etc.\ which\ often\ have\ a\ generic\ title\ only.=Дополнительный материал в "Книге" предназначен для таких элементов, как предисловия, введения, послесловия и т.д.
```
Comment (koppor) [inline src/main/resources/l10n/JabRef_ru.properties:2228]: @Siedlerchr just for info - the space is added by some crowdin magic...

image

**Review by koppor — APPROVED** [review]

File: src/main/resources/l10n/JabRef_ru.properties (hunk 5 above)
Comment (Siedlerchr) [reply src/main/resources/l10n/JabRef_ru.properties:2228]: Yes these checks have to be handled carefully

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**